CODEOWNERS

```diff
@@ -142,6 +142,7 @@
 /drivers/i2c/i2c_ll_stm32*                @ldts @ydamigos
 /drivers/i2c/i2c_rv32m1_lpi2c*            @henrikbrixandersen
 /drivers/i2c/*sam0*                       @Sizurka
+/drivers/i2c/*xec*                        @albertofloyd @scottwcpg
```

Comment:
Change to  
/drivers/*/*chhp*                           @albertofloyd @scottwcpg @franciscomunoz 
we don't have create a CODEOWNERS commit for each new driver in the future

Reply:
agreed